api/lib/use-cases/GetAttachment.js

```diff
@@ -1,11 +1,18 @@
+const mimeTypes = require('mime-types');
+
 module.exports = function(options) {
   const imageServerGateway = options.imageServerGateway;
+  const dbGateway = options.dbGateway;
 
   return async function(imageId) {
     const outputDoc = await imageServerGateway.getDocument(imageId);
-
+    const metadata = await dbGateway.getEmailAttachmentMetadata(imageId);
+    const name = metadata[0].name;
```

Comment:
does this return multiple metadata entries or is it just 1 and it's in an array? Are we sure that we always need [0]?

Reply:
just one thing in an array